centralserver/stats/models.py

```diff
@@ -1,51 +0,0 @@
-"""
-"""
-import datetime
-from annoying.functions import get_object_or_None
-
-from django.conf import settings; logging = settings.LOG
-from django.db import models
-
-from fle_utils.django_utils.classes import ExtendedModel
-from securesync.models import ID_MAX_LENGTH, IP_MAX_LENGTH
-
-
-class UnregisteredDevice(ExtendedModel):
```

Comment:
The models in this file are an important source of stats. Please don't delete them.